docs/source/diagnosis.md

```diff
@@ -0,0 +1,64 @@
+# Quantization diagnosis
```

Comment:
Diagnosis is not only for quantization but for benchmark, so this docs should include both. This is a feature introduce doc, so it's different from the blog.

Reply:
For the feature doc, we have a formal structure. 
1. We have a content in the top. 
2. The main structure will be like the following, don't need to be exactly the same title, but should mainly follow the flow. I think you already try to refer some other docs, let's polish it together.  
- Introduction( the workflow that we just created can be added here. ) 
- Supported Feature Matrix  
- Get Started  
- Examples